src/utils/FilteredTodos.ts

```diff
@@ -0,0 +1,19 @@
+import { Todo } from '../types/types/Todo';
+import { TodoStatus } from '../types/types/TodoStatus';
+
+export const filteredTodos = (todos: Todo[], filteredBy: TodoStatus) => {
+  const filteredTodos = todos.filter(todo => {
```

Comment:
```suggestion
  return todos.filter(todo => {
```
and file name should be in `camelCase` - it's not a component